Microsoft.AVS.VMFS/Microsoft.AVS.VMFS.psd1

```diff
@@ -0,0 +1,140 @@
+
+    # Script module or binary module file associated with this manifest.
+    RootModule = 'Microsoft.AVS.VMFS.psm1'
+
+    # Version number of this module.
+    ModuleVersion = '1.0.0'
+
+    # Supported PSEditions
+    # CompatiblePSEditions = @()
+
+    # ID used to uniquely identify this module
+    GUID = '1bc2c94d-907f-4184-9224-cf2bf07470af'
+
+    # Author of this module
+    Author = 'Sanjay Rajmohan'
+
+    # Company or vendor of this module
+    CompanyName = 'Microsoft Corporation'
+
+    # Copyright statement for this module
+    Copyright = '(c) Microsoft. All rights reserved.'
+
+    # Description of the functionality provided by this module
+    Description = 'Azure VMware Solutions VMFS Plugin.'
+
+    # Minimum version of the PowerShell engine required by this module
+    PowerShellVersion = '5.1'
+
+    # Name of the PowerShell host required by this module
+    # PowerShellHostName = ''
+
+    # Minimum version of the PowerShell host required by this module
+    # PowerShellHostVersion = ''
+
+    # Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+    # DotNetFrameworkVersion = ''
+
+    # Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+    # ClrVersion = ''
+
+    # Processor architecture (None, X86, Amd64) required by this module
+    # ProcessorArchitecture = ''
+
+    # Modules that must be imported into the global environment prior to importing this module
+    RequiredModules = @(
+        @{ "ModuleName" = "Microsoft.AVS.Management"; "ModuleVersion" = "5.2.39" }
+    )
+
+    # Assemblies that must be loaded prior to importing this module
+    # RequiredAssemblies = @()
+
+    # Script files (.ps1) that are run in the caller's environment prior to importing this module.
+    # ScriptsToProcess = @()
+
+    # Type files (.ps1xml) to be loaded when importing this module
+    # TypesToProcess = @()
+
+    # Format files (.ps1xml) to be loaded when importing this module
+    # FormatsToProcess = @()
+
+    # Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+    # NestedModules = @()
+
+    # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+    FunctionsToExport = @(
+        "Configure-VmfsIscsi",
+        "New-VmfsDatastore",
+        "Unmount-VmfsDatastore",
+        "Resize-VmfsVolume",
+        "Resignature-VmfsVolume"
+    )
+
+    # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+    CmdletsToExport = @(
+        "Configure-VmfsIscsi",
+        "New-VmfsDatastore",
+        "Unmount-VmfsDatastore",
+        "Resize-VmfsVolume",
+        "Resignature-VmfsVolume"
```

Comment:
Is this a valid verb?
Try `Import-Module -verbose` to see if there are any warnings

Reply:
Updated the function names to use Powershell valid verbs